File: TodoApp.Web.WebApplication/app/Model/LoginViewModel.ts
export class LoginViewModel {
    constructor() {
        this.RememberMe = false;
        this.Email = "";
        this.grant_type = "password";
    }
    public UserName: string;
    public Password: string;
    public Email: string;
    public RememberMe: boolean;
    public grant_type: string;
}

File: TodoApp.Business.Entities/ViewModels/LoginViewModel.cs
﻿using System;
using System.ComponentModel.DataAnnotations;
using System.Runtime.Serialization;

namespace TodoApp.Business.Entities.ViewModels
{
    public class LoginViewModel
    {
        [Required]
        public string UserName { get; set; }
        [Required]
        [DataType(DataType.Password)]
        public string Password { get; set; } 

        [Display(Name = "Email")] 
        public string Email { get; set; }

        [Display(Name = "Remember me?")]
        public bool RememberMe { get; set; }

    }
}
